app/components/rdf-input-fields/address-selector.hbs

```diff
@@ -0,0 +1,8 @@
+{{#if this.initialized}}
+  <Extra::AdressenregisterSelector
+    @address={{this.address}}
+    @onChange={{this.updateAddress}}
+  />
+{{else}}
+  Loading...
```

Comment:
Maybe we should use the page-loader component here?

Reply:
yes!! there is already a story for this! I think we should use skeleton loaders